js/src/controls/SplitMapControl.js

```diff
@@ -25,21 +25,11 @@ var LeafletSplitMapControlView = LeafletControlView.extend({
         LeafletSplitMapControlView.__super__.initialize.apply(this, arguments);
         this.map_view = this.options.map_view;
     },
-
-    render: function () {
+    
+    create_obj: function () {
         var that = this;
         var left_layer = this.model.get('left_layer');
         var right_layer = this.model.get('right_layer');
```

Comment:
`that` is not used, you can remove it. Also, `right_layer` and `left_layer` are defined twice in your method, but not with the same definition, I guess those three lines should be removed.